### ISSUE TITLE
Scipy 1.7.3 issue on Windows 10

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#1102]
Hi @psavery and @cjh1.  There is a hexrdgui user at UC Berkeley doing HEDM that ran into an error on startup after updating to the latest (as of today) conda packages.  I'll try to get the full stack trace for here, but essentially scipy 1.7.3 from conda-forge was throwing an error on import due to a missing dll.

Her system:
```
2:55 PM
Device name	DESKTOP-2UIVIT1
Processor	Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz   2.59 GHz
Installed RAM	16.0 GB (15.8 GB usable)
Device ID	4F6D70A8-25B8-4E3C-B15B-B976B80C8F7C
Product ID	00326-10000-00000-AA685
System type	64-bit operating system, x64-based processor
Pen and touch	Pen and touch support with 10 touch points
Edition	Windows 10 Home
Version	20H2
Installed on	‎4/‎1/‎2021
OS build	19042.1415
Experience	Windows Feature Experience Pack 120.2212.3920.0
```

**Comment by cjh1** [comment]
Is this the 0.8.14 release or a prerelease build? Stack trace could be great.

**Comment by joelvbernier** [comment]
Pre-release.  Complaining about a missing dll; can’t duplicate on my windows 11 system.

**Comment by cjh1** [comment]
Ok, I will try in my machine. What conda command is the user issuing? The one in the readme?

**Comment by joelvbernier** [comment]
this:

```
conda create --name hexrdgui-prerelease python=3.8 hexrd hexrdgui -c hexrd -c hexrd-prerelease -c hexrdgui-prerelease -c conda-forge
```

**Comment by joelvbernier** [comment]
Also @cjh1, downgrading to scipy 1.6 from the main channel resolved the issue.  Might just be a wonky build on conda-forge?  I asked the student to build that test env to get a proper stack trace.

**Comment by cjh1** [comment]
> conda create --name hexrdgui-prerelease python=3.8 hexrd hexrdgui -c hexrd -c hexrd-prerelease -c hexrdgui-prerelease -c conda-forge


The invocation needs the label path, in my environment: 

```
conda create --name hexrdgui-prerelease python=3.8 hexrd hexrdgui -c hexrd/label/hexrd -c hexrd/label/hexrd-prerelease -c hexrd/label/hexrdgui-prerelease -c conda-forge
```

In an environment created with the above command on Window 10 Pro HEXRDGUI runs without a problem.

**Comment by joelvbernier** [comment]
realized my typo, sorry!

**Comment by cjh1** [comment]
Note: The environment created for me includes `scipy-1.7.3-py38ha1292f7_0` and everything works fine. What conda version is being used?

**Comment by joelvbernier** [comment]
must be something specific to her system.  Closing; will reopen if we see it again.

**Comment by joelvbernier** [comment]
Aha -- here is the stack trace Michelle is getting.  Repeatable upon installing the pre-release hexrd/hexrdgui

```
(hexrd-prerelease) C:\Users\miche>hexrdgui
Traceback (most recent call last):
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\Scripts\hexrdgui-script.py", line 6, in <module>
    from hexrd.ui.main import main
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\hexrd\ui\main.py", line 9, in <module>
    from hexrd.ui.main_window import MainWindow
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\hexrd\ui\main_window.py", line 6, in <module>
    from hexrd.ui.image_stack_dialog import ImageStackDialog
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\hexrd\ui\image_stack_dialog.py", line 8, in <module>
    from hexrd.ui.constants import MAXIMUM_OMEGA_RANGE
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\hexrd\ui\constants.py", line 4, in <module>
    from hexrd import constants
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\hexrd\constants.py", line 31, in <module>
    from scipy import constants as sc
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\scipy\__init__.py", line 136, in <module>
    from . import _distributor_init
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\scipy\_distributor_init.py", line 59, in <module>
    WinDLL(os.path.abspath(filename))
  File "C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\ctypes\__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
FileNotFoundError: Could not find module 'C:\Users\miche\anaconda3\envs\hexrd-prerelease\lib\site-packages\scipy\.libs\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). Try using the full path with constructor syntax.
```

**Comment by joelvbernier** [comment]
Is she just missing gfortran?

**Comment by cjh1** [comment]
@joelvbernier Can you ask Michelle to try setting `CONDA_DLL_SEARCH_MODIFICATION_ENABLE=1` before starting hexrdgui? `set CONDA_DLL_SEARCH_MODIFICATION_ENABLE=1`. Looks like we might be running into [this issue](https://github.com/scipy/scipy/issues/14002). Looks like our move to 3.9 might resolve the issue too.

**Comment by joelvbernier** [comment]
What is the prognosis on moving the 3.9?

**Comment by joelvbernier** [comment]
@cjh1 -- to be clear, you mean do:
```
set CONDA_DLL_SEARCH_MODIFICATION_ENABLE=1
conda create --name hexrdgui-prerelease python=3.8 hexrd hexrdgui -c hexrd/label/hexrd -c hexrd/label/hexrd-prerelease -c hexrd/label/hexrdgui-prerelease -c conda-forge
``` 
in a PowerShell, yes?

**Comment by cjh1** [comment]
@joelvbernier  Sorry, its not needed for the environment creation, just before she runs hexrdgui, not powdershell just regular command prompt. 

```
set CONDA_DLL_SEARCH_MODIFICATION_ENABLE=1
hexrdgui
```

**Comment by joelvbernier** [comment]
Bumping this -- that didn't fix the issue.

**Comment by cjh1** [comment]
Ok, can we try that latest pre-release? Its now using Python 3.9. Which was also reported as another possible solution.

**Comment by psavery** [comment]
@joelvbernier Is this now fixed as of the latest pre-release?

**Comment by joelvbernier** [comment]
Yes!
